src/main/java/duke/CommandParser.java

```diff
@@ -0,0 +1,139 @@
+package duke;
+
+import error.*;
```

Comment:
Import classes explicitly; don't use wildcard imports

Reply:
I thought this was okay since I was using all of the classes in the error package, but yes I'll change it